packages/button/src/css/index.ts

```diff
@@ -46,23 +38,28 @@ export default {
     alignItems: 'stretch',
     padding: `${layout.spacingSmall} ${layout.spacingMedium}`,
     border: 0,
-    borderRadius: '2px',
-    fontSize: type.fontSize200,
+    borderRadius: '3px',
+    fontSize: type.fontSize300,
```

Comment:
Gray area of what "breaking change" means in UI code....

With the larger font size, do you see the buttons as wider and thus creating reflows when deployed to pages?